gtest/gtest_tree.cpp

```diff
@@ -328,379 +310,352 @@ struct ComplexParallelTest : testing::Test
         action_1 = new BT::ActionTestNode("action 1");
         condition_1 = new BT::ConditionTestNode("condition 1");
 
-
         action_2 = new BT::ActionTestNode("action 2");
         condition_2 = new BT::ConditionTestNode("condition 2");
 
-
         action_3 = new BT::ActionTestNode("action 3");
         condition_3 = new BT::ConditionTestNode("condition 3");
 
         root = new BT::ParallelNode("root", 2);
         parallel_1 = new BT::ParallelNode("par1", 3);
         parallel_2 = new BT::ParallelNode("par2", 1);
 
-        parallel_1->AddChild(condition_1);
-        parallel_1->AddChild(action_1);
-        parallel_1->AddChild(condition_2);
-        parallel_1->AddChild(action_2);
+        parallel_1->addChild(condition_1);
+        parallel_1->addChild(action_1);
+        parallel_1->addChild(condition_2);
+        parallel_1->addChild(action_2);
 
-        parallel_2->AddChild(condition_3);
-        parallel_2->AddChild(action_3);
+        parallel_2->addChild(condition_3);
+        parallel_2->addChild(action_3);
 
-        root->AddChild(parallel_1);
-        root->AddChild(parallel_2);
+        root->addChild(parallel_1);
+        root->addChild(parallel_2);
     }
 };
 
-
-
 /****************TESTS START HERE***************************/
 
-
-
 TEST_F(SimpleSequenceTest, ConditionTrue)
 {
     std::cout << "Ticking the root node !" << std::endl << std::endl;
     // Ticking the root node
-    BT::ReturnStatus state = root->Tick();
+    BT::NodeStatus state = root->tick();
 
-    ASSERT_EQ(BT::RUNNING, action->Status());
+    ASSERT_EQ(BT::RUNNING, action->status());
     ASSERT_EQ(BT::RUNNING, state);
-    root->Halt();
+    root->halt();
 }
 
-
 TEST_F(SimpleSequenceTest, ConditionTurnToFalse)
 {
-    BT::ReturnStatus state = root->Tick();
+    BT::NodeStatus state = root->tick();
     condition->set_boolean_value(false);
 
-    state = root->Tick();
+    state = root->tick();
     ASSERT_EQ(BT::FAILURE, state);
-    ASSERT_EQ(BT::HALTED, action->Status());
-    root->Halt();
+    ASSERT_EQ(BT::HALTED, action->status());
+    root->halt();
 }
 
-
 TEST_F(ComplexSequenceTest, ComplexSequenceConditionsTrue)
 {
-    BT::ReturnStatus state = root->Tick();
+    BT::NodeStatus state = root->tick();
 
-    ASSERT_EQ(BT::RUNNING, action_1->Status());
+    ASSERT_EQ(BT::RUNNING, action_1->status());
     ASSERT_EQ(BT::RUNNING, state);
-    root->Halt();
+    root->halt();
 }
 
-
 TEST_F(ComplexSequence2ActionsTest, ConditionsTrue)
 {
-    BT::ReturnStatus state = root->Tick();
+    BT::NodeStatus state = root->tick();
 
-    state = root->Tick();
+    state = root->tick();
 
-    std::this_thread::sleep_for(std::chrono::seconds(5));
-    state = root->Tick();
-    state = root->Tick();
+    std::this_thread::sleep_for(std::chrono::milliseconds(500));
```

Comment:
is that done to save time during tests?

Reply:
Yes, the only purpose. All the sleeps have been slashed to 1/10th.